index.js

```diff
@@ -129,6 +129,11 @@ function checkCommand(command, msg) {
       pokeFetch.fetchSprite(pokemon.url).then(sprites => {
         // Extract sprite and official artwork
         const spriteUrl = sprites.front_default;
+        if (!spriteUrl) {
+          console.log(`Redoing the explore because i got a ${pokemon.name} without a front sprite`);
```

Comment:
Can you alter the console output to: 

`Warning: front_default sprite for ${pokemon.name} is null. Fetching new pokemon.`

or something similar? I wanted to keep the console outputs standardized for any warnings or errors that might occur, and we use this convention to output warnings for configure.js